Sources/ComposableArchitecture/Reducer/Reducers/PresentationReducer.swift

```diff
@@ -477,8 +477,8 @@ extension Task where Success == Never, Failure == Never {
   }
 }
 extension EffectPublisher {
-  internal func _cancellable(
-    id: AnyHashable = _PresentedID(),
+  internal func _cancellable<ID: Hashable>(
+    id: ID = _PresentedID(),
```

Comment:
I think this is safe for the versions of Swift we support, but we could also create overloads...